pandas/core/internals/blocks.py

```diff
@@ -2978,6 +2978,30 @@ def where(self, other, cond, align=True, errors='raise',
                                                axis=axis, transpose=transpose)
         return result
 
+    def replace(self, to_replace, value, inplace=False, filter=None,
+                regex=False, convert=True):
+        result = self if inplace else self.copy()
```

Comment:
so rather than do this here, can you add a ``.replace()`` method on ``Categorical`` and just call it here (this by-definition is a 1-D block holder).

Reply:
hold off on this pending discussion about what belongs on EAs

Reply:
I am -1 on this PR as is; this should be a method on a Categorical.